### PR TITLE
Forward compatibility with stream 1.0, 0.7, 0.6, and 0.5

### PR DESCRIPTION
**Pull request by WyriHaximus**
As mentioned on the roadmap in #1, support the latest stream tags as well.

### CODE REVIEW

**Comment by clue** [comment]
FWIW: This does not affect BC, so we may as well release this as v0.1.1 :+1:

**Comment by WyriHaximus** [comment]
> FWIW: This does not affect BC, so we may as well release this as v0.1.1 👍

I was anticipating more breaking to be honest 🤓, 0.1.1 makes sense to me, what about you @jsor?

File: composer.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
     },
     "require": {
         "php": ">=5.3",
-        "react/stream": "^0.4 || ^0.3",
+        "react/stream": "^1.0 || ^0.7 || ^0.6 || ^0.5 || ^0.4 || ^0.3",
```
Comment (kelunik) [inline composer.json]: I wouldn't include 1.0 here, as 1.0 isn't released yet. Changes could happen that are not planned yet.

**Comment by jsor** [comment]
I'm good with either.

File: composer.json (hunk 1 above)
Comment (WyriHaximus) [inline composer.json]: Anything between `0.7` and `1.0` will be bug fixes. Nothing major will change

**Comment by WyriHaximus** [comment]
Will tag `v0.1.1` tomorrow morning 👍

**Pull request merged**